.github/CONTRIBUTING.md

```diff
@@ -1,151 +1,52 @@
-- Visual Studio 2019 Community, Professional or Enterprise
+- Visual Studio 2022 Community, Professional or Enterprise
 - .NET SDKs (https://dotnet.microsoft.com/download)
-    - .NET 4.8
-	- .NET Core 3.1 SDK 
-
-
-
-## Unit Tests
-
-Unit tests can be run in either the Visual Studio Test Exploror or via command line `dotnet test`.
-
-## Functional Tests
-It is recommended to rely on unit tests to test functionalities wherever possible. For doing end-to-end validation, functional tests exists for all the modules. Unless doing significant changes,
-it is not required to run func-tests locally. All the tests, including functional tests, are automatically run when a PR is submitted.
-
-These tests works like described below:
-
-Functional tests contain test apps which refers to the product dlls from the local build. Tests deploy the Test apps to IIS/Docker and http requests are fired against it to trigger various scenarios.
-Tests apps are modified to send telemetry to a fake ingestion endpoint controlled by tests. Tests then validate the telemetry received by this endpoint.
-
-### Pre-requisites:
-
-To execute the functional tests, you need to install some additional prerequisites:
-
-- For **Web** and **PerformanceCollector** tests IIS Express should be installed.
-		
-- For **Dependency Collector**, you need to install Docker for windows as these tests need several additional dependencies to be deployed like SQL Server and Azure Emulator. 
-		Docker for Windows (https://docs.docker.com/docker-for-windows/install/). 		
-		After installation switch Docker engine to Windows Containers.(https://blogs.msdn.microsoft.com/webdev/2017/09/07/getting-started-with-windows-containers/)
-		And finally, make sure you can run ```docker run hello-world``` successfully to confirm that your machine is Docker ready.
-				
-### Running functional tests:
-
-Before running the functional tests, the product code should be built following 'Build' instructions above.
-
-After building, open the respective solutions from locations given below, build the solution. Tests will appear in Visual Studio Test Explorer and can be run from there.
-
-The following solutions contains the functional tests for various features.
-
-"\Test\Web\FunctionalTests.sln" -- Functional tests using apps onboarded with the nuget Microsoft.ApplicationInsights.Web
-Helper script to build product and run all tests in this solution - ```runFunctionalTestsWeb```
-
-"..\bin\Debug\Test\Web\FunctionalTests" -- Binary location for Test and Test apps.
-
-"\Test\PerformanceCollector\FunctionalTests.sln" -- Functional tests using apps onboarded with the nuget Microsoft.ApplicationInsights.PerfCounterCollector
-Helper script to build product and run all tests in this solution - ```runFunctionalTestsPerfCollectorAndQuickPulse```
+    - .NET Framework 4.5.2
+	- .NET Framework 4.6.2
+	- .NET Framework 4.7.2
+	- .NET Framework 4.8
+	- .NET Core 3.1
+	- .NET 5
```

Comment:
do we need .NET ? its out-of-support already.

Reply:
it's still needed for now because some of the Test projects are targeting it.

Reply:
for a separate PR, it'd be nice to remove them. (Unless some scenarios like auto-instrumentation needs it..)